pycbc/io/live.py

```diff
@@ -310,6 +319,14 @@ def save(self, fname):
             logging.info('P_astro file saved as %s', self.pastro_file)
         return
 
+        # Save EMBright properties info as json
+        if  self.hasmassgap is not None:
+            self.embright_file = self.basename + '_em_bright.json'
+            with open(self.embright_file, 'w') as embrightf:
+                json.dump({'HasMassGap': self.hasmassgap}, embrightf)
```

Comment:
To be compatible with the schema of the expected 'em bright' file you should also add keys 'HasNS' and 'HasRemnant'.  The values for these are still to be determined (we do not have very clear guidance from low latency yet) so I would set the values to -1 in order to be clear that they are not valid probabilities.

Reply:
Here is the reasoning - json does not have a None or NaN or null object, so we would have to use "None" or whatever string. But this would probably break whatever parsing is in gwcelery assuming float values. I think we don't want to make people rewrite gwcelery parsing, but we do want them to think about switching between using pipeline values if valid, vs using the 'standard' calculation if not valid.

Reply:
Then you want the other fields not to be in the .json, i.e. no change in the current proposed code. In the absence of clear guidance from LL this may be the best short-term course.